tests/lcb/stdlib/system.lcb

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright (C) 2017 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+module com.livecode.system.tests
+
+use com.livecode.system
+use com.livecode.__INTERNAL._testlib
+
+constant kValidArchs is ["x86", "x86_64", "i386", "arm", "arm64"]
```

Comment:
@livecodeali `i386` is no longer valid and `` is.

Reply:
thanks